microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -21,40 +21,86 @@ In your performance recordings, some of these long-running tasks may be **Recalc
+
+To view aggregate statistics of the CSS rule selectors that are involved in the entire performance recording:
+
+1. Deselect any event that's selected in the performance recording, by clicking an empty area of the flame chart.
+
+1. Select the entire recording range. To do this, double-click in the **CPU** chart at the top of the **Performance** tool or, using your mouse, hover over the flame chart and scroll up until the entire chart is displayed.
+
+1. In the bottom section of the **Performance** tool, click **Selector Stats**:
+
+   ![The Selector Stats table for the full recording](./selector-stats-images/full-recording.png)
 
 
 <!-- ====================================================================== -->
 ## Analyze Selector Stats
 
-You can sort the data in the **Selector Stats** table in ascending or descending order for any of the columns by clicking the column header.  For example, sort the **Elapsed (µs)** column in descending order to see which selectors take up the most time in the **Recalculate Style** event you have selected.
+To sort the data that's displayed in the **Selector Stats** table in ascending or descending order, click a column header.  For example, to see which selectors take up the most time, click the **Elapsed (µs)** column header:
+
+![The Selector Stats table with the selectors sorted by elapsed time, in descending order](./selector-stats-images/sort-by-elapsed-time.png)
+
+If you find a CSS selector that took the browser a long time to calculate and was matched many times, it's a good candidate to try and improve.  Try to change your selector so it requires less time to calculate and matches fewer elements on the page.  How to improve your CSS selectors depend on your particular use case. Repeat the steps from the previous section to confirm that your changes helped decrease the **Recalculate Style** event duration.
 
-As a general rule, if you find a CSS selector that took the browser a long time to calculate and was matched many times, that's a good candidate to try and improve.  Try to change your selector so it requires less time to calculate and matches fewer elements on the page.  How to improve your CSS selectors depend on your particular use case. Repeat the steps from the previous section to confirm that your changes helped decrease the **Recalculate Style** event duration.
 
 <!-- ====================================================================== -->
 ## See also
 
 * [Get started analyzing runtime performance](./index.md) - An introduction to analyzing runtime performance in DevTools.
 * [Performance features reference](./reference.md) - Covers many of the features in the **Performance** tool.
-* [The truth about CSS selector performance](https://blogs.windows.com/msedgedev/2023/01/17/the-truth-about-css-selector-performance/)
\ No newline at end of file
+* [The truth about CSS selector performance](https://blogs.windows.com/msedgedev/2023/01/17/the-truth-about-css-selector-performance/)
```

Comment:
```suggestion
* [The truth about CSS selector performance](https://blogs.windows.com/msedgedev/2023/01/17/the-truth-about-css-selector-performance/) - Windows blog post.
```
gives more uniform result, from POV of gh rendered.

gh rendered section has no "external" icon indicator:
https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/selector-stats/microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md#see-also